spec/serializers/provider_serializer_spec.rb

```diff
@@ -35,4 +35,33 @@
   it { should include(postcode: provider.enrichments.last.postcode) }
   it { should include(institution_type: provider.provider_type) }
   it { should include(accrediting_provider: nil) }
+
+  describe 'ProviderSerializer#region_code' do
+    subject do
+      serialize(provider)["region_code"]
+    end
+
+    describe "provider region code 'London' can be overriden by enrichment region code 'Scotland'" do
+      let(:enrichment) do
+        build(:provider_enrichment, region_code: "Scotland")
+      end
+
+      let(:provider) { create :provider, region_code: "London", enrichments: [enrichment] }
+      it { is_expected.not_to eql(format("%02d", 1)) }
+      it { is_expected.to eql(format("%02d", 11)) }
+      it { expect(subject.length).to eql(2) }
```

Comment:
Just some minor niggles here:

- `format("%02d", 11)` can be `'%02d' % 11`, which in turn can simply just be `'11'`.
- likewise for `'01'`
- testing the length is already done with `is_expected.to eql '11'`